tests/e2e/run-scanner-v4.bats

```diff
@@ -70,6 +74,258 @@ teardown() {
     run remove_existing_stackrox_resources
 }
 
+# We are using our own deploy function, because we want to have the flexibility to patch down resources
+# after deployment. Without this we are only able to special-case local deployments and CI deployments,
+# but not, for example, manual testing on Infra.
+#
+# shellcheck disable=SC2120
+_deploy_stackrox() {
+    local tls_client_certs=${1:-}
+    local central_namespace=${2:-stackrox}
+    local sensor_namespace=${3:-stackrox}
+
+    deploy_stackrox_operator
+
+    _deploy_central "${central_namespace}"
+    export_central_basic_auth_creds
+    wait_for_api "${central_namespace}"
+    setup_client_TLS_certs "${tls_client_certs}"
+    record_build_info "${central_namespace}"
+
+    _deploy_sensor "${sensor_namespace}" "${central_namespace}"
+    echo "Sensor deployed. Waiting for sensor to be up"
+    sensor_wait "${sensor_namespace}"
+
+    # Bounce collectors to avoid restarts on initial module pull
+    "${ORCH_CMD}" -n "${sensor_namespace}" delete pod -l app=collector --grace-period=0
+
+    sensor_wait "${sensor_namespace}"
+
+    wait_for_collectors_to_be_operational "${sensor_namespace}"
+
+    touch "${STATE_DEPLOYED}"
+}
+
+# shellcheck disable=SC2120
+_deploy_central() {
+    local central_namespace=${1:-stackrox}
+    info "Deploying central to namespace ${central_namespace}"
+
+    # If we're running a nightly build or race condition check, then set CGO_CHECKS=true so that central is
+    # deployed with strict checks
+    if is_nightly_run || pr_has_label ci-race-tests || [[ "${CI_JOB_NAME:-}" =~ race-condition ]]; then
+        ci_export CGO_CHECKS "true"
+    fi
+
+    if pr_has_label ci-race-tests || [[ "${CI_JOB_NAME:-}" =~ race-condition ]]; then
+        ci_export IS_RACE_BUILD "true"
+    fi
+
+    if [[ "${DEPLOY_STACKROX_VIA_OPERATOR}" == "true" ]]; then
+        deploy_central_via_operator "${central_namespace}"
+    else
+        if [[ -z "${OUTPUT_FORMAT:-}" ]]; then
+            if pr_has_label ci-helm-deploy; then
+                ci_export OUTPUT_FORMAT helm
+            fi
+        fi
+
+        DEPLOY_DIR="deploy/${ORCHESTRATOR_FLAVOR}"
+        CENTRAL_NAMESPACE="${central_namespace}" "${ROOT}/${DEPLOY_DIR}/central.sh"
+    fi
```

Comment:
Can't this be replaced by a call to `deploy_central()`?

Reply:
Yes, I will push a simplifying commit.